pjsip/src/pjsua-lib/pjsua_call.c

```diff
@@ -644,6 +644,7 @@ PJ_DEF(void) pjsua_call_setting_default(pjsua_call_setting *opt)
     pj_bzero(opt, sizeof(*opt));
     opt->flag = PJSUA_CALL_INCLUDE_DISABLED_MEDIA;
     opt->aud_cnt = 1;
+    opt->custom_call_id = pj_str("");
```

Comment:
minor optimization: unnecessary? as the `bzero()` above should set `pj_str_t.len` to zero already.

Reply:
Removed this. It appears unnecessary in my tests.